app/views/articles/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @article.title %></h1>
+
+<p><%= @article.body %></p>
+
+<ul>
+  <li><%= link_to "Edit", edit_article_path(@article) %></li>
+  <li>
+    <%= link_to "Destroy", article_path(@article), method: :delete, data: { confirm: "Are you sure?" } %>
```

Comment:
What is the purpose of this option `method: :delete` here?

Reply:
sends as parameter the method which will be used in controller, also for create the url which will be understood by `routes.rb`